elyra/metadata/metadata_app.py

```diff
@@ -473,6 +473,96 @@ def start(self):
               ("is" if num_invalid_exported == 1 else "are") + " invalid)")
 
 
+class SchemaspaceImport(SchemaspaceBase):
+    """Handles the 'import' subcommand functionality for a specific schemaspace."""
+
+    directory_option = CliOption("--directory", name='directory',
+                                 description='The local file system path from where the metadata will be imported',
+                                 required=True)
+
+    replace_flag = Flag("--replace", name='replace',
+                        description='Overwrite existing metadata instance with the same name',
+                        default_value=False)
+
+    # 'Import' flags
+    options: List[Option] = [directory_option, replace_flag]
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.metadata_manager = MetadataManager(schemaspace=self.schemaspace)
+
+    def start(self):
+        super().start()  # process options
+
+        directory = self.directory_option.value
+
+        src_directory = os.path.join(directory, self.schemaspace)
```

Comment:
I don't think we should require that the instances to be imported are in a schemaspace specific directory - even though they may have been exported that way.  I believe this will be confusing to users and users may have their instances (obtained via a tar file or some other means) sitting in a (relatively) random directory.  I think we should import directly from whatever `--directory` specifies.